ml-data-hub-plugin/build.gradle

```diff
@@ -49,6 +49,8 @@ repositories {
     mavenCentral()
     maven { url 'http://repo.spring.io/milestone' }
     maven { url 'http://developer.marklogic.com/maven2' }
+    //ToDo: Remove this once java-client-api-5.3.2 is released
+    maven {url 'http://distro.marklogic.com/nexus/repository/maven-releases/'}
```

Comment:
I don't think this is needed here and web/build.gradle  but it's going to be deleted, so I think it's fine for now.

Reply:
If I don't add the repo, QS is not getting compiled.